Source/SwiftyDropbox/Shared/Generated/Sharing.swift

```diff
@@ -6472,6 +6477,8 @@ open class Sharing {
         case insideOsxPackage
         /// We do not support sharing the Vault folder.
         case isVault
+        /// We do not support sharing the Family folder.
+        case isFamily
```

Comment:
not sure if this Family folder thing could impact us

Reply:
These generated files are for their service integration. This looks like a new error added for shared family folders which impact the `shareFolder` and `unshareFolder` apis which we do not use.

Reply:
thanks for looking into it 🙏